man/man8/zfs.8

```diff
@@ -2986,6 +3066,103 @@ except that the default types to display are
 .Fl t Sy posixgroup Ns \&, Ns Sy smbgroup .
 .It Xo
 .Nm
+.Cm projectspace
+.Op Fl Hp
+.Oo Fl o Ar field Ns Oo , Ns Ar field Oc Ns ... Oc
+.Oo Fl s Ar field Oc Ns ...
+.Oo Fl S Ar field Oc Ns ...
+.Ar filesystem Ns | Ns Ar snapshot
+.Xc
+Displays space consumed by, and quotas on, each project in the specified filesystem or snapshot. This subcommand is identical to
+.Nm zfs Cm userspace ,
+except that the project identifier is numeral, not name. So need neither the option
+.Sy -i
+for SID to POSIX ID nor
+.Sy -n
+for numeric ID, nor
+.Sy -t
+for types.
+.It Xo
+.Nm
+.Cm project
```

Comment:
As described above `zfs_do_project` there are 4 mutually exclusive ways of executing this sub-command.  This situation is handled in the rest of the man page by adding a section for each use case, again just like above `zfs_do_project`.  Please break it out the same way here for consistency and to more importantly to prevent confusion regarding its usage.  For options which are applicable the multiple sections go ahead and duplicate them.

```
zfs project -p id file|directory...
  Set project ID...

  -p ...

zfs project -c [-dr0] [-p id] file|directory...
  Check the...  

  -c ...
  -d ...
  -p ...
  -r ...
  -0 ...

zfs project -C [-rk] file|directory...
  Clear the...

  -C ...
  -k ...
  -r ...

zfs project -s [-r] [-p id] file|directory...
  Set up a project hierarchy...

  -p ...
  -r ...
  -s ...
```